src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -120,13 +118,13 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
    * of custom tip, including zero.
    */
   minimumTipPercentage = 1 as const;
-  readonly suggestedTipPercentages = {
-    '7.5': '7.5%',
-    '10': '10%',
-    '12.5': '12.5%',
-    '15': '15%',
-    'Other': 'Other'
-  };
+  readonly suggestedTipPercentages = [
+    {value: '7.5', label: '7.5%'},
+    {value: '10', label: '10%'},
+    {value: '12.5', label: '12.5%'},
+    {value: '15', label: '15%'},
+    {value: 'Other', label: 'Other'}
+  ];
```

Comment:
this should really type error until the new version of components is pulled in. I might see if I can find a way to make it do that.

Reply:
it displays undefined atm

Reply:
Right - it can't work until the components PR is merged and the new version installed here. But I'd rather it had a build error instead of displaying undefined.